src/components/VotingOptionTag/VotingOptionTag.story.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import type { Hst } from '@histoire/plugin-svelte';
+	import VotingOptionTag from './VotingOptionTag.svelte';
+	import { DefaultVoteOption } from '$models/voting';
+	export let Hst: Hst;
+</script>
+
+<Hst.Story title="VotingOptionTag">
+	<div class="flex flex-col gap-4">
+		<div>
+			<VotingOptionTag voteOption={DefaultVoteOption.Agreed} />
```

Comment:
I suggest splitting each variation into separated `Hst.Variant` and using a "grid" layout ([see more in the doc](https://histoire.dev/guide/svelte3/stories.html#variants)) to make it more obvious what is the single unit of the component.